boilerplate.js

```diff
@@ -192,13 +203,7 @@ async function install(context) {
 
     await system.spawn(`ignite add ${boilerplate} ${debugFlag}`, { stdio: "inherit" })
 
-    // react native link -- must use spawn & stdio: ignore or it hangs!! :(
-    spinner.text = `▸ linking native libraries`
-    spinner.start()
-    await system.spawn("react-native link", { stdio: "ignore" })
-    spinner.stop()
-
-    await ignite.addModule("react-native-gesture-handler", { version: "1.1.0", link: true })
+    await ignite.addModule("react-native-gesture-handler", { version: "1.1.0", link: false })
```

Comment:
We could probably update this to `1.1.3` which is the latest version. Also, `link` [defaults to falsy](https://github.com/infinitered/ignite/blob/4807df913565945aba6d23e1f82230a543970de0/src/extensions/ignite/add-module.ts#L53), so we can just remove that option.

```suggestion
    await ignite.addModule("react-native-gesture-handler", { version: "1.1.3" })
```

Reply:
Thanks for pointing this out -- we've already got react-native-gesture-handler in package.json (at version 1.30 now!); I think that auto linking means that the boilerplate doesn't need to mention this at all, so I'll remove it (and retest!).

Reply:
I love removing code. :tada: